### ISSUE TITLE
Drawers not working on Dedicated/Realms Server

### ISSUE THREAD

**Issue by Pezguru** [jaquadro/StorageDrawers#570]
When playing on a Dedicated or Realms server the drawers recipes will not work, in creative the blocks are shown, you can put them into you inventory and try to place but when placed it disappears where you placed the block.  If you open your inventory the block disappears from there.  Everything works fine on a single or LAN based game.  No other mods are failing. 

Tried on 1.12 & 1.12.1
Forge 14.21.12387

**Comment by jaquadro** [comment]
The mod needs to be installed on the server, but aren't realms servers by definition unmoddable server-side?

**Comment by Pezguru** [comment]
Didn't know this was a server side mod.  Not sure on the realms serer but
but a server from Havoc so hopefully i can do server side mods on that.
Thanks

**Comment by Pezguru** [comment]
I installed forge-1.12.1-14.22.1.2478-installer-win,
Chameleon-1.12-4.1.3, StorageDrawers-1.12.1-5.3.2 on the server.  It loads
but the drawers will not come up a recipe but you can access them in
creative. Once placed they disappear right away.

**Comment by smbarbour** [comment]
After installing Forge and the mods to the server, make sure that you are starting the server using the Forge jar and not the vanilla server jar, otherwise Forge and the other mods will not be loaded.

**Comment by Pezguru** [comment]
Yeah I did that. I have other mods that will work but something about this
one is not showing the recipes or allow them to be placed.
